app/styles/project/_c-playground.scss

```diff
@@ -40,3 +40,11 @@
   overflow-x: hidden;
   background-color: $au-gray-100;
 }
+
+.playground__tab {
```

Comment:
Nitpick, I would rename the class to `.playground-tab`. It's not a child element of the `playground` block, so it feels weird to use the element notation here.